### PR TITLE
Update port selection after initialization

### PR DESCRIPTION
**Pull request by ASDosjani**
After the start of the configurator the msp selector or the manual port box didn't show up. This line fixes it.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/port_handler.js
Comment (haslinghuis) [inline src/js/port_handler.js:165]:
```suggestion
            GUI.updateManualPortVisibility();
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
This way we don't interfere with firmware flasher tab change function. As the change function is also called in serial_backend.js

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
O wait, why did you change all occurrences?

File: src/js/port_handler.js
Comment (ASDosjani) [reply src/js/port_handler.js]: I replaced the other similar lines as well.

Comment (haslinghuis) [reply src/js/port_handler.js]: image

On line 8, maybe revise the function to use toggle

**Comment by ASDosjani** [comment]
> O wait, why did you change all occurrences ?
> 
> Please only update the original line 165. The other lines are used in firmware flasher tab.

Sorry, I thought they are doing the same thing.

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
